service/github/github.go

```diff
@@ -323,12 +323,22 @@ func (g *PullRequest) diffUsingGitCommand(ctx context.Context) ([]byte, error) {
 
 	headSha := pr.GetHead().GetSHA()
 
-	mergeBaseBytes, err := exec.Command("git", "merge-base", headSha, pr.GetBase().GetSHA()).Output()
+	commits, _, err := g.cli.Repositories.CompareCommits(ctx, g.owner, g.repo, headSha, pr.GetBase().GetSHA(), nil)
 	if err != nil {
-		return nil, fmt.Errorf("failed to get merge-base commit: %w", err)
+		return nil, err
+	}
+	if commits == nil || commits.MergeBaseCommit == nil || commits.MergeBaseCommit.SHA == nil {
+		return nil, errors.New("cannot to get merge base commit")
```

Comment:
failed to get merge base commit

Reply:
I apply it: https://github.com/reviewdog/reviewdog/pull/1714/commits/0a67a71027994b5fb1c167f31d5347e1c7171e00